CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+* [FEATURE] Call RUM's `errorEventMapper` for crashes.
 * [FEATURE] Support calling log event mapper for crashes.
```

Comment:
nit/ Let's add PR references to both to be consistent with the rest of this file. It helps when generating release notes and later when tracking changes.